docker-compose.yml

```diff
@@ -44,6 +44,8 @@ services:
       EQ_RABBITMQ_HOST_SECONDARY: rabbit
       EQ_SECRETS_FILE: docker-secrets.yml
       EQ_KEYS_FILE: docker-keys.yml
+      EQ_ENVIRONMENT_PREFIX: dev
+      EQ_DYNAMODB_ENDPOINT: http://localhost:6060
```

Comment:
you should route the communication within the docker network by changing this to `http://eq-docker-dynamodb:6060`